src/sharepoint/sharepoint.controller.js

```diff
@@ -62,6 +64,31 @@ angular
             });
         }
 
+        editDisplayName () {
+            this.displayName = this.sharepoint.displayName || this.sharepoint.domain;
+            this.editMode = true;
+        }
+
+        saveDisplayName () {
+            const displayName = this.displayName || null;
```

Comment:
Does it really change anything if the  `displayName` is `undefined` ? If not, this line is useless

Reply:
it's important for the API